Can't install cabal-hoogle

When I try to install cabal-hoogle using last master with ghc-9.2.7 I get following error:

```
$ cabal install exe:cabal-hoogle --overwrite-policy=always
Wrote tarball sdist to
/Users/al.makarov/Programming/cabal-hoogle/dist-newstyle/sdist/cabal-hoogle-3.10.0.0.tar.gz
Resolving dependencies...
Build profile: -w ghc-9.2.7 -O1
In order, the following will be built (use -v for more details):
 - cabal-install-3.10.1.0 (lib) (requires build)
 - conduit-1.3.5 (lib) (requires build)
 - warp-tls-3.3.6 (lib) (requires build)
 - conduit-extra-1.3.6 (lib) (requires build)
 - http-conduit-2.3.8.2 (lib) (requires build)
 - hoogle-5.0.18.3 (lib) (requires build)
 - cabal-hoogle-3.10.0.0 (lib) (requires build)
 - cabal-hoogle-3.10.0.0 (exe:cabal-hoogle) (requires build)
Starting     warp-tls-3.3.6 (lib)
Starting     conduit-1.3.5 (lib)
Starting     cabal-install-3.10.1.0 (lib)
Building     cabal-install-3.10.1.0 (lib)
Building     warp-tls-3.3.6 (lib)
Building     conduit-1.3.5 (lib)
Installing   conduit-1.3.5 (lib)
Completed    conduit-1.3.5 (lib)
Installing   cabal-install-3.10.1.0 (lib)
Completed    cabal-install-3.10.1.0 (lib)

Failed to build warp-tls-3.3.6.
Build log ( /Users/al.makarov/.cabal/logs/ghc-9.2.7/wrp-tls-3.3.6-53f50343.log
):
Configuring library for warp-tls-3.3.6..
Preprocessing library for warp-tls-3.3.6..
Building library for warp-tls-3.3.6..
[1 of 2] Compiling Network.Wai.Handler.WarpTLS.Internal ( Network/Wai/Handler/WarpTLS/Internal.hs, dist/build/Network/Wai/Handler/WarpTLS/Internal.o, dist/build/Network/Wai/Handler/WarpTLS/Internal.dyn_o )
[2 of 2] Compiling Network.Wai.Handler.WarpTLS ( Network/Wai/Handler/WarpTLS.hs, dist/build/Network/Wai/Handler/WarpTLS.o, dist/build/Network/Wai/Handler/WarpTLS.dyn_o )

Network/Wai/Handler/WarpTLS.hs:389:40: error:
    • Couldn't match type ‘x509-1.7.7:Data.X509.CertificateChain.CertificateChain’
                     with ‘crypton-x509-1.7.6:Data.X509.CertificateChain.CertificateChain’
      Expected: Maybe
                  crypton-x509-1.7.6:Data.X509.CertificateChain.CertificateChain
        Actual: Maybe
                  x509-1.7.7:Data.X509.CertificateChain.CertificateChain
      NB: ‘crypton-x509-1.7.6:Data.X509.CertificateChain.CertificateChain’
            is defined in ‘Data.X509.CertificateChain’
                in package ‘crypton-x509-1.7.6’
          ‘x509-1.7.7:Data.X509.CertificateChain.CertificateChain’
            is defined in ‘Data.X509.CertificateChain’ in package ‘x509-1.7.7’
    • In the ‘tlsClientCertificate’ field of a record
      In the first argument of ‘return’, namely
        ‘TLS
           {tlsMajorVersion = major, tlsMinorVersion = minor,
            tlsNegotiatedProtocol = proto,
            tlsChiperID = TLS.cipherID infoCipher,
            tlsClientCertificate = clientCert}’
      In a stmt of a 'do' block:
        return
          TLS
            {tlsMajorVersion = major, tlsMinorVersion = minor,
             tlsNegotiatedProtocol = proto,
             tlsChiperID = TLS.cipherID infoCipher,
             tlsClientCertificate = clientCert}
    |
389 |               , tlsClientCertificate = clientCert
    |                                        ^^^^^^^^^^
Error: cabal: Failed to build warp-tls-3.3.6 (which is required by
exe:cabal-hoogle from cabal-hoogle-3.10.0.0). See the build log above for
details.
```

--- a/cabal.project
+++ b/cabal.project
@@ -1,1 +1,2 @@
 packages: ./
+index-state: 2023-06-05T00:00:00Z

